roles/zabbix_agent/tasks/configure-Windows.yml

```diff
@@ -79,4 +79,8 @@
   ansible.windows.win_service:
     name: "{{ zabbix_agent_win_service }}"
     state: started
-    start_mode: delayed
+    start_mode: "{{ zabbix_agent_service_start_mode | default('delayed') }}"
+    failure_actions: "{{ zabbix_agent_service_failure_actions is defined | ternary(zabbix_agent_service_failure_actions, omit) }}"
```

Comment:
I think you should use defalut filter like a zabbix_agent_service_start_mode paramter.

```yaml
failure_actions: "{{ zabbix_agent_service_failure_actions | default(omit) }}"
```

Reply:
Yep, you're right :+1: Not sure why I thought I needed to check for defined.